src/main/java/org/opensearch/knn/index/util/Lucene.java

```diff
@@ -73,4 +74,9 @@ public float score(float rawScore, SpaceType spaceType) {
         // score provided.
         return rawScore;
     }
+
+    @Override
+    public List<String> mmapFileExtensions() {
+        return List.of("vec", "vem");
```

Comment:
Why not "vex" file?

Reply:
let me correct, it should be "vex" instead of "vem", we don't need "vem" as it has only metadata and doesn't influence latencies as per my tests. Good catch Jack!